docs/src/main/paradox/grpc-replicated-event-sourcing-transport.md

```diff
@@ -0,0 +1,146 @@
+# Akka Replicated Event Sourcing over gRPC
+
+Akka Replicated Event Sourcing extends Akka Persistence allowing multiple replicas of the same entity, all accepting
+writes, for example in different data centers or cloud provider regions. This makes it possible to implement patterns 
+such as active-active and hot standby. 
+
+Originally, Akka Replicated Event Sourcing has required cross-replica access to the underlying replica database, which
+can be hard to open up for security and infrastructure reasons.
```

Comment:
Maybe also clarify that the replicas don't have to be part of the same Akka Cluster? The original didn't require that but in practise it was our recommendation to use it with Akka multi-dc Cluster, which is not needed for this.

Reply:
Added some rambling in a56def842eda0ee4de801549976dde745d8bafc6